tests/roles/backend_services/tasks/main.yaml

```diff
@@ -123,6 +123,20 @@
   delay: 5
 # TODO- debug value (was 2)
 
+- name: verify that MariaDB and RabbitMQ CR's deployed, for all defined cells
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc get {{ item }} -o json | jq -e '[ .items[].status.conditions[] | select(.type == "Ready") | .message] | select(length > 0)  | all(. == "Setup complete")'
+  register: mariadb_cr_result
```

Comment:
small nit :) , this can be something like _service__cr_result instead of "mariadb" since you are looping over mariadb and rabbit , and sometime in the future people might add more services in this loop